Octokit.Tests.Integration/Reactive/ObservableEventsClientTests.cs

```diff
@@ -617,7 +618,82 @@ public TheGetAllUserPerformedPublicMethod()
 
         public class TheGetAllForAnOrganizationMethod
         {
-           
+            readonly ObservableEventsClient _eventsClient;
+            readonly string _organization;
+            readonly string _user;
+
+            public TheGetAllForAnOrganizationMethod()
+            {
+                var github = Helper.GetAuthenticatedClient();
+                _eventsClient = new ObservableEventsClient(github);
+                _user = github.User.Current().Result.Login;
+                _organization = github.Organization.GetAllForCurrent().Result.First().Login;
+            }
+
```

Comment:
I don't believe the test failure is related to this PR.

Here's what I can see: https://github.com/octokit/octokit.net/issues/1251